arc: add print sub-command and JWK in header support to verify (#47)
veraison/ear: arc/README.md

* synthesising attestation results in EAR (EAT Attestation Result) format,
* cryptographically verifying and displaying the contents of an EAR

> * printing EAR header and payload without verifying

Reviewer: ```suggestion
* printing the EAR header and payload without verification
```

| `--color` | trustworthiness vector report colourises the tiers (default is B&W) |
| `<jwt-file>` | a JWT wrapping an EAR claims-set |

> If `--pkey` parameter is omitted or the default file name is specified then
> * key from the file will be used if exists ignoring keys in JWT header
> * the public key and algorithm from JWT header will be used if the file is missing

Reviewer: ```suggestion
If the `--pkey` parameter is omitted or the default file name is specified, the key from the file will be used if it exists, ignoring the keys in the JWT header.
Instead, if the file is missing, the public key and algorithm from the JWT header will be used.
```

* The EAR claims-set is printed to stdout.
* If present, the _decoded_ trust vector is also printed to stdout (the exact format depends on `--verbose` and `--color`).

## Print

The `print` sub-command is used to print the contents of a EAR, including the header.

> No ERA validation or veryfing are executed.

Reviewer: ```suggestion
Neither EAR validation nor verification is executed.
```
